deeplake/integrations/mmdet/mmdet_.py

```diff
@@ -459,29 +460,34 @@ def format_results(self, results, jsonfile_prefix=None, **kwargs):
 
 
 class HubDatasetCLass:
-    def __init__(self, cfg=None, ds=None, tensors=None):
+    def __init__(self, cfg=None, ds=None, tensors=None, mode="train"):
         if ds:
             self.ds = ds
+            tensors = tensors or {}
         else:
-            creds = cfg.get("deeplake_credentials", {})
+            creds = cfg.data[mode].get("deeplake_credentials", {})
```

Comment:
@adolkhan @farizrahman4u Just checking if this make sense. It feels like the only cfg element that should be needed here is the cfg.data, not the whole cfg